server/src/schema.js

```diff
@@ -24,10 +34,19 @@ type Mutation {
 }
 
 type Subscription {
-  taskCreated: Task,
-  taskModified: Task,
-  taskDeleted: ID
+  taskMutated: TaskMutationPayload!
 }
+
+type TaskMutationPayload {
+  mutation: MutationType!
+  task: Task!
+}
+enum MutationType {
```

Comment:
This can be improved further over the time by introduction of modules! 
Great stuff.